tests/playwright-test/playwright.expect.true.spec.ts

```diff
@@ -322,3 +322,41 @@ test('should print syntax error', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(1);
   expect(result.output).toContain(`Unexpected token "]" while parsing selector "row]"`);
 });
+
+test('should support toBeOK', async ({ runInlineTest, server }) => {
+  const result = await runInlineTest({
+    'a.test.ts': `
+      const { test } = pwt;
+
+      test('pass with response', async ({ page }) => {
+        const res = await page.request.get('${server.EMPTY_PAGE}');
+        expect(res).toBeOK();
```

Comment:
You need await there afaik. Otherwise it ends up in an unhanded promise rejection.

Reply:
Yeah, good point.